api/finetune/v1beta1/finetune_types.go

```diff
@@ -59,7 +70,47 @@ type Hyperparameter struct {
 	HyperparameterRef string `json:"hyperparameterRef"`
 	// Overrides is used to override some hyperparameter
 	// +kubebuilder:validation:Optional
-	Overrides map[string]string `json:"overrides,omitempty"`
+	Overrides *Parameters `json:"overrides,omitempty"`
+}
+
+type Parameters struct {
+	// Scheduler specifies the learning rate scheduler.
+	// +kubebuilder:validation:Enum=Cosine;Linear;Constant
+	Scheduler HyperparameterScheduler `json:"scheduler,omitempty"`
+	// Optimizer specifies the optimization algorithm.
+	// +kubebuilder:validation:Enum=AdamW;Adam;SGD
+	Optimizer HyperparameterOptimizer `json:"optimizer,omitempty"`
+	// Int4 indicates whether to use 4-bit integer quantization.
+	Int4 bool `json:"int4,omitempty"`
```

Comment:
this field should be `*bool`

Reply:
Yeah, you're right. :kissing_smiling_eyes: